### PR TITLE
DEVED-3590: Upgrade to modern versions

### PR DESCRIPTION
**Pull request by patillacode**
Laravel and PHP versions have been updated.
Minor syntax and functionality rewrites.
Local testing.

### CODE REVIEW

**Comment by patillacode** [comment]
I still need to remove some unnecessary files and add the travis config file.
I will do this on my following commits, although if you are reviewing/testing this should not matter at all.

File: app/AppointmentReminders/AppointmentReminder.php [hunk 1]
```diff
@@ -48,6 +48,7 @@ function ($appointment) {
      */
     private function _remindAbout($appointment)
     {
+        echo "I am actually sending something!!!!!";
```
Comment (maylonpedroso) [inline app/AppointmentReminders/AppointmentReminder.php]: @patillacode this `echo` here looks like a debug message?

**Comment by maylonpedroso** [comment]
@patillacode , is tests config ok? looks like the CI is not running any test (https://travis-ci.org/TwilioDevEd/appointment-reminders-laravel/builds/485425299#L702)

File: app/AppointmentReminders/AppointmentReminder.php (hunk 1 above)
Comment (patillacode) [reply app/AppointmentReminders/AppointmentReminder.php]: Yep, thanks!
Changed.

**Comment by patillacode** [comment]
Thanks @maylonpedroso 
I have added the tests under the correct folder structure for the upgraded _laravel_ version
Now tests are triggering correctly.

https://travis-ci.org/TwilioDevEd/appointment-reminders-laravel/builds/485838696#L695

**Pull request merged**